### PR TITLE
Feature/recording

### PR DESCRIPTION
**Pull request by cphung1**
Still need to actually display results

### CODE REVIEW

File: screens/PracticeScreen.js [hunk 1]
```diff
@@ -265,9 +265,11 @@ const styles = StyleSheet.create({
   continueButtonActive: {
     backgroundColor: Styles.SECONDARY_COLOR,
     width: "90%",
+    marginBottom: 20
   },
   continueButtonInactive: {
     backgroundColor: Styles.GREY_COLOR,
     width: "90%",
+    marginBottom: 20
```
Comment (cphung1) [inline screens/PracticeScreen.js:273]: oops prev comment about the marginBottom was for this

Comment (cphung1) [inline screens/PracticeScreen.js:273]: oops comment about the marginBottom was for this

File: models/Speech.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Platform } from "react-native";
 import * as FileSystem from 'expo-file-system';
 
 const BACKEND_URL = "https://conversational-backend.herokuapp.com";
-const PRONUNCIATION_BACKEND_URL = "https://f50f9d64.ngrok.io";
+const PRONUNCIATION_BACKEND_URL = "http://53d19ad5.ngrok.io";
```
Comment (kevwang1) [inline models/Speech.js:9]: Change this to "http://production.getvivoce.com:8000"  - @xmaayy deployed the server

File: models/Speech.js [hunk 3]
```diff
@@ -104,10 +105,20 @@ async function evaluateSpeech(audioStrBase64, target, suggestions) {
   const finalResults = Object.assign({
     speechToText: results[1]
   }, results[0]);
-  console.log(finalResults);
+  console.log("Final Results:", finalResults);
   return finalResults;
 }
 
+export function correctSpeechToText(userSpeech) {
```
Comment (kevwang1) [inline models/Speech.js:112]: Do you use this anywhere?

File: screens/PracticeScreen.js [hunk 4]
```diff
@@ -29,9 +33,13 @@ export default class PracticeScreen extends React.Component {
       isRecording: false,
       wordFeedback: {},
       wordFeedbackCardVisible: false,
-      // For listening-based exercises
```
Comment (kevwang1) [inline screens/PracticeScreen.js:32]: Add this line back

File: screens/PracticeScreen.js [hunk 5]
```diff
@@ -29,9 +33,13 @@ export default class PracticeScreen extends React.Component {
...
+      progress: 0,
       selectedWordIdx: -1
     };
+    // this.state.exercises = props.route.params.exercises;
+    this.soundObject = new Audio.Sound();
+    Permissions.askAsync(Permissions.AUDIO_RECORDING);
+      // For listening-based exercises
     this.state.exercises = JSON.parse(JSON.stringify(props.route.params.exercises)); // Make a copy
     this.soundObject = new Audio.Sound();
```
Comment (kevwang1) [inline screens/PracticeScreen.js:44]: I think this is duplicated now

File: screens/PracticeScreen.js (hunk 1 above)
Comment (kevwang1) [reply screens/PracticeScreen.js:273]: Seems to still look good for me

File: screens/PracticeScreen.js [hunk 6]
```diff
@@ -29,9 +33,13 @@ export default class PracticeScreen extends React.Component {
       isRecording: false,
       wordFeedback: {},
       wordFeedbackCardVisible: false,
-      // For listening-based exercises
+      progress: 0,
       selectedWordIdx: -1
     };
+    // this.state.exercises = props.route.params.exercises;
```
Comment (kevwang1) [inline screens/PracticeScreen.js:39]: You can delete this line now

File: models/Speech.js (hunk 3 above)
Comment (cphung1) [reply models/Speech.js:112]: nope forgot to take that out

**Pull request merged**